src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/ManageOffencesService.kt

```diff
@@ -15,4 +16,12 @@ class ManageOffencesService(
   fun getSdsExclusionsForOffenceCodes(offenceCodes: List<String>): List<SDSEarlyReleaseExclusionForOffenceCode> {
     return manageOffencesApiClient.getSdsExclusionsForOffenceCodes(offenceCodes)
   }
+
+  fun getOffences(offenceCodes: Set<String>): List<Offence> {
```

Comment:
Is this method used anywhere? if not required can it be removed + anything associated, ege Offence model definition

Reply:
Its not used now Manage Offences is updated to provide TORERA offence codes. I'll remove the code as suggested.